source/neuropod/internal/neuropod_tensor.cc

```diff
@@ -33,13 +33,13 @@ std::vector<int64_t> compute_strides(const std::vector<int64_t> &dims)
     std::vector<int64_t> out(dims.size());
 
     int64_t running_product = 1;
-    for (int i = out.size() - 1; i >= 0; i--)
+    for (size_t i = out.size(); i > 0; i--)
```

Comment:
interesting way to avoid overflow. I usually used either ssize_t (this is from C10 I guess) or long long in for loop and use the same approach with negative i at the end.

Reply:
If I used a signed type (like `int` or `ssize_t`), I'd need to do static casts within the loop to avoid additional warnings. I'm okay with either solution.

Do you think the static cast solution is more clear?